packages/teleterm/src/ui/DocumentCluster/ClusterResources/Databases/Databases.tsx

```diff
@@ -83,7 +86,8 @@ function ConnectButton({
     <Cell align="right">
       <MenuLoginTheme>
         <MenuLogin
-          placeholder="Enter username"
+          {...getMenuLoginOptions(protocol)}
+          width="195px"
```

Comment:
As much as I dislike setting a specific width to elements, I can't really think of a better solution here. ;f